src/components/site/checkpoint/index.jsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
 import svg from './checkpoint.svg';
 import { checkpointContainer } from './checkpoint.module.scss';
 
 const Checkpoint = () => {
+  const { t } = useTranslation('interface');
```

Comment:
in functional components `t()` can be pulled from `useTranslation()` instead of using a HOC